srcpkgs/python3-hkdf/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'python3-hkdf'
+pkgname=python3-hkdf
+version=0.0.3
+revision=1
+archs=noarch
+wrksrc=hkdf-${version}
+build_style=python3-module
+hostmakedepends="python3 python3-setuptools"
+short_desc="HMAC Key Derivation module"
+maintainer="travankor <travankor@tuta.io>"
+license="BSD-2-Clause"
+homepage="https://pypi.org/project/hkdf/"
+distfiles="https://files.pythonhosted.org/packages/c3/be/327e072850db181ce56afd51e26ec7aa5659b18466c709fa5ea2548c935f/hkdf-${version}.tar.gz"
```

Comment:
use PYPI_SITE, try hashless address like in python3-autobahn

Reply:
switched it to debian mirror for now

Reply:
"${PYPI_SITE}/h/hkdf/hkdf-${version}.tar.gz" works